05.wc/wc.rb

```diff
@@ -0,0 +1,76 @@
+# ! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+SHOW_ADJUST_SPACE_SIZE = 1
+STANDARD_INPUT_EXCEPTIONAL_WORD = '-'
+TOTAL = '合計'
+
+def main
+  options = parse_options(ARGV)
+  counts = []
+  ARGF.each(nil) do |standard_input_or_file|
+    count = {}
+
+    count[:filename] = ARGF.filename
+    count[:line] = standard_input_or_file.lines.count if options[:l]
+    count[:word] = standard_input_or_file.split(/\s+/).size if options[:w]
+    count[:bytesize] = standard_input_or_file.size if options[:c]
+    counts << count
+  end
+  rows = counts
+  # 複数行出力する場合は、集計行を表示する
+  rows << get_wc_total_row(counts) if counts.size > 1
+  max_column_widths = get_max_column_widths(rows)
+  show_wc_rows(rows, max_column_widths)
+end
+
+def get_wc_total_row(counts)
+  total_row = counts.inject({}) do |result, column_name|
+    result.merge(column_name) { |_key, current_val, adding_value| current_val + adding_value }
+  end
+  total_row[:filename] = TOTAL
+  total_row
+end
+
+def show_wc_rows(rows, max_column_widths)
+  rows.each do |row|
+    cells = {}
+    row.each_pair do |column_name, cell|
```

Comment:
```suggestion
    row.each do |column_name, cell|
```

each_pair を使う人って僕の周りでは見たことがないので、eachの方がいいと思います。
（僕もeach_pairってなんだっけ？と思って調べてしまった）